pages/property-filter/token-editor.scss

```diff
@@ -0,0 +1,17 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+// Imitate popover container
+.token-editor-container {
+  max-inline-size: 912px;
```

Comment:
Can we make this a variable? This feels like a very random value without any context.

Reply:
This is defined for test pages only, I'd keep it like that rather, at least until the popover size for the new token editor is figured out.